app/views/public/share.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="section">
+  <div class="title is-3">Find a way to share</div>
+</div>
+<div class="section">
+  <div id="public-listings">
+  </div>
+</div>
+
+<script>
+    document.addEventListener('DOMContentLoaded', () => {
+        EntryPoints.publicListings('#public-listings')
```

Comment:
added this endpoint that's rendering publicListings until we move that code under the PublicController#share